.docstr.yaml

```diff
@@ -0,0 +1,14 @@
+paths:
+  - datastacks
+  - de_workloads
+  - pysparkle
+  - utils
+exclude: .*/tests # regex
+verbose: 3 # int (0-4)
+skip_magic: True # Boolean
+skip_file_doc: True # Boolean
+skip_init: True # Boolean
+skip_class_def: False # Boolean
+skip_private: True # Boolean
+accept_empty: True # Boolean
+fail_under: 80 # int
```

Comment:
Good spot ;-) So I have intentionally left out the ones that needed adding under silver and pysparkle, due to the impending refactor - otherwise no reason we can't aim for 100% (but can adjust that after the refactor)?

Reply:
yeah, it's ok for now, but it would be good to increase as we go :) resolving